bartender/pyrabbit.py

```diff
@@ -28,12 +27,27 @@ def is_alive(self):
             return False
 
     def verify_virtual_host(self):
+        """Ensure the virtual host exists"""
         try:
             return self._client.get_vhost(self._virtual_host)
         except Exception:
             self.logger.error("Error verifying virtual host %s, does it exist?", self._virtual_host)
             raise
 
+    def ensure_admin_expiry(self):
+        """Ensure that the admin queue expiration policy exists"""
+        try:
+            kwargs = {
+                'pattern': '^admin.*',
```

Comment:
`admin` seems like a very common name that people might have. If I read this correctly, if someone has their own `admin` queue, then we will expire that on their behalf. Do you think maybe we should prepend `bg` to our admin queue names so that we can at least have a little bit of deniability? Oh wait, nevermind, I just remembered you can set the virtualhost.